packages/connect/src/backend/Blockchain.ts

```diff
@@ -88,6 +86,11 @@ export class Blockchain {
     }
 
     onError(error: ERRORS.TrezorError) {
+        const pendingSubscriptions =
+            this.link.listenerCount('block') ||
+            this.link.listenerCount('notification') ||
+            this.link.listenerCount('fiatRates');
```

Comment:
just a side note - I can imagine, that in the future, there might be a new subscribtion type a we forget to  list it here. But I can't come up with a good solution how to avoid it or at least get notified somehow

Reply:
Hmmm, true. The problem is that there are two types of events (apart from these `connected`, `disconnected` etc.) so we cannot check just any listener. :thinking:

Reply:
yes, have no clue about something generic and not breaking at the same time. so lets keep it